### DISCUSSION TITLE
好棒的项目！

### DISCUSSION THREAD

**Discussion by Timemm** [charliedream1/ai_quant_trade#3] [General]
最近也正打算本地搭个服务来着。一来是想研究下AI策略，因其对资源要求高，市面上的quant平台无法满足，二来是因为策略安全性问题，总是寄希望quant平台人员都是君子好像也不太合适哈哈。初步看了一下整体，发现因子要自己构建，thurse获取的数据都是些原始数据，如果常用的因子库也能加入愿景就更好了，毕竟市面上的quant平台几乎都有自己的因子库，而重复构建因子也是挺麻烦的。

**Comment by charliedream1** [comment]
首先谢谢对本项目的支持，您说的这些都在规化当中，后续逐步加入



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年2月4日(周六) 晚上7:14
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [charliedream1/ai_quant_trade] 好棒的项目！ (Discussion #3)




 
最近也正打算本地搭个服务来着。一来是想研究下AI策略，因其对资源要求高，市面上的quant平台无法满足，二来是因为策略安全性问题，总是寄希望quant平台人员都是君子好像也不太合适哈哈。初步看了一下整体，发现因子要自己构建，thurse获取的数据都是些原始数据，如果常用的因子库也能加入愿景就更好了，毕竟市面上的quant平台几乎都有自己的因子库，而重复构建因子也是挺麻烦的。
